pytrip/vdx.py

```diff
@@ -791,33 +791,74 @@ def get_2d_projection_on_basis(self, basis, offset=None):
 
     def get_2d_slice(self, plane, depth):
         """
-        Gets a 2d Slice object from the contour in either sagittal or coronal plane.
+        Gets a 2D Slice object from the contour in either sagittal or coronal plane.
```

Comment:
It would be great if the docstring would list how the contour points are ordered for each plane (Z,X) (X,Z) (Z,Y) or (Y,Z).
(Also great if we can mark the funtions if they can handle non-equidistant slices, and non-continous slices, which may appear in DICOM.)

Reply:
Good idea, we will add it !
This is important, and I suspect in some places still broken, see https://github.com/pytrip/pytrip/issues/593